mantidproject/mslice
# Option to toggle grid lines

This PR adds options to display the `x-` and `y-grid` lines on plots, from the `Plot Options` menu. The `x-` and `y-`lines can be toggled separately.

**To test:**

Run MSlice and check that if checked in the `Plot Options` the grid lines are displayed. 


Fixes #192.
